docs/ref/contrib/admin/index.txt

```diff
@@ -2827,6 +2827,36 @@ linked to the document in ``{% block dark-mode-vars %}``.
 
 .. _prefers-color-scheme: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
 
+.. _extra_body:
+
+``extra_body`` template block
+=============================
+
+You can define any custom code by overriding the ``{% extra_body %}`` block in
+any templates that extend ``admin/base.html``. This block will output code just
+above the closing ``</body>`` tag.
+
+Usage
+-----
+
+To override the ``extra_body`` block, include the following in your template:
+
+.. code-block:: html+django
+
+    {% extends 'admin/base.html' %}
+
+    {% block extra_body %}
+        {{ block.super }}
+        ...
+    {% endblock %}
+
+By extending the ``admin/base.html`` template and overriding the ``extra_body``
+block, you can insert any custom HTML, JavaScript, or other content that needs
+to appear just before the closing ``</body>`` tag. The ``{{ block.super }}``
+ensures that any existing content in the ``extra_body`` block from the parent
+template is preserved.
+
```

Comment:
```suggestion
The ``{{ block.super }}`` ensures that any existing content in the ``extra_body``
block from the parent template is preserved.
```
I would then remove this bit to have less duplication (there was also an extra new line that wasn't needed).